components/contribution-flow/index.js

```diff
@@ -571,10 +571,10 @@ class ContributionFlow extends React.Component {
             return false;
           } else if (!isNil(tier?.availableQuantity) && stepDetails.quantity > tier.availableQuantity) {
             return false;
-          } else if (stepDetails.platformContribution && stepDetails.platformContribution > stepDetails.amount) {
+          } else if (stepDetails.platformContribution && stepDetails.platformContribution / stepDetails.amount >= 0.5) {
```

Comment:
From the `noPaymentRequired` variable above, I assume that `stepDetails.amount` can be `=== 0` here (=> division by 0). I don't think tips are enabled for free tickets, but to be on the safe side I would change that to something like:

```suggestion
          } else if (stepDetails.platformContribution && (!stepDetails.amount || stepDetails.platformContribution / stepDetails.amount >= 0.5)) {
```

Reply:
Can I instead only do the check if `stepDetails.amount` is truthy?

Reply:
Yes, that's what the suggestion above does:

```es6
!stepDetails.amount || stepDetails.platformContribution / stepDetails.amount >= 0.5
```

Reply:
Ok, I wrote it differently.